requirements/test.txt

```diff
@@ -1,4 +1,4 @@
 mock>=1.3.0
 nose==1.3.7
-PyHamcrest==1.9.0
+PyHamcrest
```

Comment:
same here.

Reply:
Hi,
changed this to version 2.1.0, in my next commit!